metricbeat/module/kubernetes/state_statefulset/state_statefulset.go

```diff
@@ -50,10 +49,6 @@ var (
 			"statefulset": p.KeyLabel("name"),
 			"namespace":   p.KeyLabel(mb.ModuleDataKey + ".namespace"),
```

Comment:
Here is the magic I mentioned above. The part I don't know how you can access this label data below. You probably have to fetch it and delete the key and we change the "framework" later?